src/views/SystemIntake/Review/index.tsx

```diff
@@ -59,14 +53,17 @@ const Review = ({ systemIntake, now }: ReviewProps) => {
       </Button>
       <Button
         type="submit"
-        disabled={isSubmitting}
+        disabled={mutationResult.loading}
         onClick={() =>
-          dispatch(
-            postAction({
-              actionType: 'SUBMIT_INTAKE',
-              intakeId: systemIntake.id
-            })
-          )
+          mutate({
+            variables: {
+              input: { id: systemIntake.id }
+            }
+          }).then(response => {
+            if (!response.errors) {
```

Comment:
What happens if there are errors?  Is that handled elsewhere?

Reply:
We don't currently have error handling on these pages, but I think we can add some!